application/org.openjdk.jmc.feature.core/feature.xml

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--   
    Copyright (c) 2018, 2022, Oracle and/or its affiliates. All rights reserved.
+   Copyright (c) 2022, Datadog, Inc. All rights reserved.
```

Comment:
We don't add copyright lines to already existing files. Only when creating new files.